backend/interfaces/IJobData.ts

```diff
@@ -0,0 +1,59 @@
+import { JobState } from 'enums/JobState';
```

Comment:
In a coming ticket, we should move all the types into a root-level `shared` directory.

Reply:
Yes.  I'm starting to run into that too...